my_enumerable.rb

```diff
@@ -0,0 +1,13 @@
+module Enumerable
+  def all(list)
+    list.all? { |num| num < 5 }
+  end
+
+  def any(list)
```

Comment:
A method that returns a bool should have `?` at the end so it should be defined as `any?` not `any`.
`any?` should return `true` if any element meets a specified condition and false otherwise. For example if the list = [1, 2, 3, 4], then `list.any? {|e| e == 2}` returns `true`.